config/environments/test.rb

```diff
@@ -4,6 +4,12 @@
   Env.set_for_development_and_test!
   ENV['ENABLE_CLASS_LISTS'] = 'true'
 
+    # For use with studentinsights-dev Google account
```

Comment:
If this is the same in dev and test, let's put it in `Env.set_for_development_and_test!`  Maybe we don't need this for automated tests right now though?

Reply:
Since we're not going with automated tests for this feature anyway we can probably remove all of this.